app/core/src/navigation/HomepageStack.js

```diff
@@ -18,5 +19,22 @@ export default StackNavigator(
   {
     ...StackNavigatorOptions,
     initialRouteName: 'Home',
+    headerMode: 'none',
+  },
+);
+
+export default StackNavigator(
+  {
+    Main: {
+      screen: HomeStack,
+    },
+    LocationPicker: {
+      screen: withMappedProps(LocationPicker),
```

Comment:
@tbergq This is only temporary, right? Will this be part of the hotels' navigation stack?

Reply:
Yes, that was the idea, when it is all done, we connect it to hotels navigation stack